NARFO_BE/NARFO_BE/Startup.cs

```diff
@@ -27,10 +31,37 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddDbContext<narfoContext>(opt =>
-                opt.UseInMemoryDatabase("narfo"));
+            var connection = @"Server=dev.retrotest.co.za;Initial Catalog=narfo;Persist Security Info=False;User ID=group2;Password=jtn8TVNQMW_28esy;";
+            services.AddDbContext<narfoContext>
+               (options => options.UseSqlServer(connection));
+
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-        }
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.AddCors(o => o.AddPolicy("MyPolicy", builder =>
+            {
+                builder.AllowAnyOrigin()
+                       .AllowAnyMethod()
+                       .AllowAnyHeader();
+            }));
+            services.AddSwaggerGen(c =>
```

Comment:
you are using swagger?! create proper request and response models please  and also include proper descriptions of each api to not get lost in your own code 
```
// DELETE api/values/5
/// <summary>
/// Delete API Value
/// </summary>
/// <remarks>This API will delete the values.</remarks>
/// <param name="id"></param>
[HttpDelete("{id}")]
public void Delete(int id)
{
}
```
and example i got from [https://www.talkingdotnet.com/add-swagger-to-asp-net-core-2-0-web-api/]